models/battery.py

```diff
@@ -1,12 +1,18 @@
 from models.base import Model
 from typing import override
+from esr_loss_model import ESRBatteryLossModel
 
 class SCPBatteryModel(Model):
```

Comment:
I would rename this to BatteryModel as it's not derived solely form equations in the Solar Car Primer at all.